contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -1118,7 +1164,49 @@ namespace eosiosystem {
           * @param bytes - the amount of ram to sell in bytes.
           */
          [[eosio::action]]
-         void sellram( const name& account, int64_t bytes );
+         action_return_sellram sellram( const name& account, int64_t bytes );
```

Comment:
Why not `uint64_t` used here and in other places so you don't need to do the negativeness check later?